src/state/timing.ml

```diff
@@ -0,0 +1,20 @@
+
+(** Timing tree node. *)
+type tree = {
+  name: string; (** Name of node. *)
+  mutable cputime: float; (** Accumulated CPU time in seconds. *)
+  mutable walltime: float; (** Accumulated wall time in seconds. *)
+  mutable allocated: float; (** Accumulated allocated memory in bytes. *)
+  mutable count: int; (** Number of repetitions. Only set if {!Timing.countCalls} is true. *)
+  mutable children: tree list; (** Child nodes. *)
+}
+
+let create_tree name =
+  {
+    name = name;
+    cputime = 0.0;
+    walltime = 0.0;
+    allocated = 0.0;
+    count = 0;
+    children = [];
+  }
```

Comment:
Why is this here? There should be no need to duplicate it because Gobview can depend on Goblint modules. `Timing` is in a separate dune library, so adding a dependency should be enough.

Reply:
Yes, you are right. This is actually something I still had in mind to improve, just forgot to do it before setting this PR to ready. I will update this.